composer.json

```diff
@@ -23,8 +23,8 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "doctrine/orm": ">=2.2.1,<2.7-dev",
-        "doctrine/dbal": ">=2.2.1,<2.7-dev",
+        "doctrine/orm": ">=2.2.1,<3.0-dev",
+        "doctrine/dbal": ">=2.2.1,<2.8-dev",
```

Comment:
For reference, see [`composer.json`](https://github.com/doctrine/dbal/blob/1f91c2d8bb18d6f439b2dea8cab5d58ca7468c77/composer.json#L39-L43) of `doctrine/dbal:dev-master`.